config.d

```diff
@@ -0,0 +1,69 @@
+void main(const string[] args)
```

Comment:
Maybe a short header of what this script does and that its executed as part of the build process to update the VERSION file?

Reply:
Done.